src/lib/_imports/components/c-card.css

```diff
@@ -104,7 +104,7 @@
 
 /* Elements */
 :--c-card > a:not(:--c-button) {
-    display: inline-block; /* FAQ: for actions that do not yet have this */
+    display: flex;
```

Comment:
This change seems okay. But I worry about the switch away from `inline-`. I wonder whether there was a reason for `inline`, and thus whether this should use `inline-flex` instead of `flex`.

I'll look into this tomorrow or Friday. Just letting you know why I haven't merged yet. I got thinkin' to do.

Reply:
Wes, your worry is valid, but the effect is irrelevant, because of how designers use such a link in a card.

<details>

- By using `inline-` anything (e.g. `inline-block`, `inline-flex`, `inline-grid`) the code allows multiple such links on one line.
- By **not** using `inline-` anything (e.g. `block`, `flex`, `grid`) the code allows only one such link per line.

Designers only ever use this link as one per line. (They allow multiple buttons per line). So, unexpectedly, **not** using `inline-` actually supports designer usage.

</details>